### ISSUE TITLE
Autocomplete not working in webpack+vue

### ISSUE THREAD

**Issue by Enethion** [Dogfalo/materialize#5101]
## Expected Behavior

Autocomplete should be available at point of initiation

## Current Behavior

`Uncaught TypeError: $(...).autocomplete is not a function` is logged even though method is called after loading of all vendor scripts.

## Possible Solution

I've seen issue #3444 and tried everything to use this method with my vue component, but with no success so far. Any other materialize method seems to work inside vue components but not `.autocomplete()`. 
## Steps to Reproduce (for bugs)

1. Two files: `vendor.js` and `app.js` built with webpack. First one contains jquery, vue, axios and materialize scripts. Second contains all app scripts with various components. 
2. `vendor.js` is first one loaded on the page
3. One of app components is `autocomplete-input` which should be text input with autocomplete plugin on it. Plugin initialized with code below:
```
//...
  mounted(){
    $(this.$el) // perfectly point to `input` element, tested with `.hide()`
    .autocomplete({
      data: {
        'Microsoft': null,
        'Google': null
      }
    })
  },
//...
```

## Context

I was trying to make reusable vue component to keep DRY code.

## Your Environment

* Version used: 0.100.1 (according to npm)
* Browser Name and version: Google Chrome 60
* Operating System and version (desktop or mobile): Windows 10 x64

**Comment by tomscholz** [comment]
The issue template is meant to be filled out!

**Comment by Enethion** [comment]
Updated

**Comment by Enethion** [comment]
**Another update**

No idea which element caused it, but it finally works. I've updated both node and npm, which caused some build errors so I had to remove `node_modules` directory and reinstall all packages. After that, I reverted code so inside my `mounted () { ... }` method I have also `$(function () { ... })` in which I call `autocomplete` on my element, and now it works. Before update and reinstall same code I have now was invalid as well.
